### PR TITLE
V1.3.0

### PR DESCRIPTION
**Pull request by jkbonfield**
Preparation for v1.3.0 htscodecs release.

No tag has been added yet, hence no release, but `make distcheck` builds and validates the tarballs.

### CODE REVIEW

**Comment by daviesrob** [comment]
Regrettably, it looks like autotools doesn't like the concept of `NEWS.md`:
```
make[2]: *** No rule to make target 'NEWS', needed by 'distdir-am'.  Stop.
make[2]: Leaving directory '/tmp/cirrus-ci-build'
make[1]: *** [Makefile:614: distdir] Error 2
```
I don't know if it's possible to change the filename it's after?

You also have the rockylinux problem, which can be solved by changing the docker repo used to `rockylinux:9`.

**Comment by jkbonfield** [comment]
I'm struggling to work out how my "make distchest" worked before!   Current automake does claim to support adding .md suffixes where needed, but not on the system I normally use.  Maybe I ran it on a more modern system.   Anyway it's now done explicitly.

Thanks

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+  Implementations exists for SSE4.1, AVX2, AVX512 and ARM Neon, as
+  well as traditional non-SIMD scalar code in C and JavaScript. See
+  the commit logs for benchmarks.
+
+- Improved memory allocation via a new htscodecs_tls_alloc function.
+  This uses Thread Local Storage (TLS) to avoid multiple malloc/free
+  calls, reducing system CPU time.
+
+- Some external functions have been renamed, with the old ones still
+  exsiting in a deprecated fashion.  Every symbol should now start
```
Comment (daviesrob) [inline NEWS.md:19]:
```suggestion
  existing in a deprecated fashion.  Every symbol should now start
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+- Updated the Appveyor CI image to user a newer gcc.  Also added ARM
+  to the list of processors to test on.
+
+- Tab vs space code changes.  Use "git diff -w" to see through these.
+
+- Reworked fuzzing infrastructure.
+
+- Small speed improvements to various rANS encoders and decoders.
+  These were tested on a broad range of compilers, versions and
+  systems.  The new code may be slightly slower with some cominations,
```
Comment (daviesrob) [inline NEWS.md:34]:
```suggestion
  systems.  The new code may be slightly slower with some combinations,
```

**Review by daviesrob — COMMENTED** [review]
I spotted a couple of typos.  I don't know if you want to change the date on the top?

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,55 @@
+Release 1.3.0: 19th July 2022
+-----------------------------
+
+The primary change in this release is a new SIMD enabled rANS codec.
+
+Changes
+
+- There is a 32-way unrolled rANS implementation.  This is accessed
+  using the existing rans 4x16 API with the RANS_ORDER_X32 bit set.
+  Implementations exists for SSE4.1, AVX2, AVX512 and ARM Neon, as
```
Comment (jmarshall) [inline NEWS.md:10]:
```suggestion
  Implementations exist for SSE4.1, AVX2, AVX512 and ARM Neon, as
```

**Comment by jkbonfield** [comment]
Thanks both.

Clearly I need to work out how to permanently enable that emacs mode to do auto-spell checking.

**Pull request merged**